hack/generator/pkg/astmodel/package_import_set_test.go

```diff
@@ -297,24 +297,24 @@ func TestPackageImportSet_ResolveConflicts_GivenImplicityNamedConflicts_AssignsE
 			"Import conflicts with versioned resolution (i)",
 			createSet(emailTestRef, networkTestRef, emailTestAltRef, networkTestAltRef),
 			emailTestRef,
-			"emailv20180801",
+			"emailV20180801",
```

Comment:
Same comment as above -- I think we're not expecting any uppercase characters here given it's a package name?